docs/index.md

```diff
@@ -59,12 +59,22 @@ to interact with Tachyon, e.g. copy data in and out of the file system.
 # User Documentation
 
 **Deployment Guide:**
-
+<!---
```

Comment:
Alright, should I roll back most of the commits other than the improvement to the navigation loop?